frontend/src/components/LandingPage/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Card, /*CardImg, CardText,*/ CardBody,
+  CardTitle, CardSubtitle, /*Button*/
+} from 'reactstrap';
+import Image from 'react-bootstrap/Image';
```

Comment:
Jut wondering what made you choose to use the react-bootstrap image instead of reactstraps?

Reply:
I wanted to make the image of the card round, but I wasn't able to figure out how to do that with the card image element. I also didn't see any Image component on reactstrap (https://reactstrap.github.io/components/alerts/), so I used bootstrap after looking online

Reply:
I don't think you need react-bootstrap. I took out react-bootstrap Image and used card image. The image of the card was still a circle.

Reply:
This wasn't working for me, but I can try again